mail_reply_stage/tests/test_mail_reply_stage.py

```diff
@@ -0,0 +1,153 @@
+        from .test_models import TestMailReply, TestMailReplyParent, TestMailReplyStage
+
+        cls.loader.update_registry(
+            (TestMailReplyParent, TestMailReply, TestMailReplyStage)
+        )
+        cls.test_model = cls.env.ref("mail_reply_stage.model_test_mail_reply")
+        cls.parent_stage_ids_field = cls.env["ir.model.fields"]._get(
+            "test.mail.reply.parent", "stage_ids"
+        )
+        cls.parent_id_field = cls.env["ir.model.fields"]._get(
+            "test.mail.reply", "parent_id"
+        )
+        cls.reply_stage_id_field = cls.env["ir.model.fields"]._get(
+            "test.mail.reply", "stage_id"
+        )
+        cls.stage_a, cls.stage_a_xmlid = cls.creat_stage("Stage A")
+        cls.stage_b, cls.stage_b_xmlid = cls.creat_stage("Stage B")
+        cls.stage_c, cls.stage_c_xmlid = cls.creat_stage("Stage C")
+        cls.stage_d, cls.stage_d_xmlid = cls.creat_stage("Stage D")
+        cls.parent_1 = cls.env["test.mail.reply.parent"].create(
+            {
+                "name": "Test Parent 1",
+                "stage_ids": [
+                    Command.set([cls.stage_a.id, cls.stage_b.id, cls.stage_c.id])
+                ],
+            }
+        )
+        cls.parent_2 = cls.env["test.mail.reply.parent"].create(
+            {
+                "name": "Test Parent 2",
+                "stage_ids": [
+                    Command.set([cls.stage_a.id, cls.stage_b.id, cls.stage_c.id])
+                ],
+            }
+        )
+        cls.record_1 = cls.env["test.mail.reply"].create(
+            {
+                "name": "Test 1",
+                "parent_id": cls.parent_1.id,
+                "stage_id": cls.stage_a.id,
+            }
+        )
+        cls.record_2 = cls.env["test.mail.reply"].create(
+            {
+                "name": "Test 2",
+                "parent_id": cls.parent_2.id,
+                "stage_id": cls.stage_a.id,
+            }
+        )
+        cls.mail_reply_config_1 = cls.env["mail.reply.config"].create(
+            {
+                "model_id": cls.test_model.id,
+                "parent_field_id": cls.parent_id_field.id,
+                "parent_stage_field_id": cls.parent_stage_ids_field.id,
+                "domain": "[('parent_id.name', '=', 'Test Parent 1')]",
+                "reply_stage_field_id": cls.reply_stage_id_field.id,
+                "reply_stage_xml_id": cls.stage_b_xmlid.id,
+            }
+        )
+        cls.mail_reply_config_2 = cls.env["mail.reply.config"].create(
+            {
+                "sequence": 20,
+                "model_id": cls.test_model.id,
+                "parent_field_id": cls.parent_id_field.id,
+                "parent_stage_field_id": cls.parent_stage_ids_field.id,
+                "reply_stage_field_id": cls.reply_stage_id_field.id,
+                "reply_stage_xml_id": cls.stage_c_xmlid.id,
+            }
+        )
+        cls.user = cls.env["res.users"].create(
```

Comment:
```suggestion
        cls.user = cls.env["res.users"].with_context(no_reset_password=True).create(
```
or just create a partner record instead of a user.

Reply:
Done!